nimbus/src/nimbus.idl

```diff
@@ -20,12 +20,16 @@ dictionary EnrolledExperiment {
     string branch_slug;
 };
 
-dictionary ExperimentConfig {
+dictionary Config {
```

Comment:
Since we're renaming things...is there anything in here that's not related to remote-settings? If not, it may be helpful to rename this to `RemoteSettingsConfig` or similar. It would help set caller expectations in the short-term, and may encourage us to factor the code in a way that would work sensibly with a future external remote-settings client in the longer term.

Reply:
(This would also have the benefit of helping disambiguate "bucket" in the remote-settings sense and "bucket" in the experimentation sense, for the "bucket name" field)